distributed_shampoo/utils/tests/shampoo_preconditioner_list_test.py

```diff
@@ -728,22 +735,13 @@ class ShampooPreconditionerListTest(AbstractTest.BaseShampooPreconditionerListTe
     def _amortized_computation_properties(self) -> InverseRootProperties:
         return InverseRootProperties()
 
-    def _instantiate_preconditioner_list(  # type: ignore[override]
-        self, **kwargs: Any
-    ) -> ShampooPreconditionerList:
-        kwargs = {
-            "beta2": 1.0,
-            "epsilon": 0.0,
-            "use_bias_correction": True,
-            "preconditioner_config": DefaultShampooConfig,
-        } | kwargs
-        return ShampooPreconditionerList(
-            block_list=self._block_list,
-            state=self._state,
-            block_info_list=self._block_info_list,
-            factor_matrix_dtype=torch.float64,
-            **kwargs,  # type: ignore[arg-type]
-        )
+    @property
+    def _default_preconditioner_config(self) -> ShampooPreconditionerConfig:
+        return DefaultShampooConfig
+
+    @property
+    def _preconditioner_list_factory(self) -> Callable[..., PreconditionerList]:
```

Comment:
The return type of `_default_preconditioner_config` is adapted for each case preconditioner list test, but not for `_preconditioner_list_factory`. Is there a reason for this?

Reply:
To be fair, adapting the return type would be more precise, but not sure if `mypy` would complain.

Reply:
OK, let's go for this then.